packages/lib/services/search/SearchEngine.ts

```diff
@@ -68,6 +61,15 @@ export interface Terms {
 	body: (string | ComplexTerm)[];
 }
 
+export enum SearchType {
+	Auto = 'auto',
+	Basic = 'basic',
+	Nonlatin = 'nonlatin',
+	Fts = 'fts',
+}
+
+const quoted = (s: string) => s.startsWith('"') && s.endsWith('"');
```

Comment:
`isQuoted` for clarity, otherwise it looks like the function is meant to add quotes